### PR TITLE
Fluent validation added

### PR DESCRIPTION
**Pull request by holsardor**

### CODE REVIEW

File: Services/UpdateHandler.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public async Task HandleUpdateAsync(ITelegramBotClient botClient, Update update,
         {
             dbContext = scope.ServiceProvider.GetRequiredService<IAppDbContext>();
             messageLocalizer = scope.ServiceProvider.GetRequiredService<IStringLocalizer<Resources.Message>>();
-            controlLocalizer = scope.ServiceProvider.GetRequiredService<IStringLocalizer<Resources.Control>>();
+            controlLocalizer = scope.ServiceProvider.GetRequiredService<IStringLocalizer<Resources. Control>>();
```
Comment (wahid-d) [inline Services/UpdateHandler.cs:45]: Neag o'zgardi bu qator

File: Dtos/UpdateProductDto.cs [hunk 2]
```diff
@@ -2,6 +2,7 @@ namespace DurgerKing.Dtos;
 
 public class UpdateProductDto
 {   
+
```
Comment (wahid-d) [inline Dtos/UpdateProductDto.cs:5]: nega?

File: Controllers/ProductsController.cs [hunk 3]
```diff
@@ -12,14 +12,14 @@ namespace DurgerKing.Controllers;
...
-
+    
     public ProductsController(IAppDbContext dbContext)
     {
         this.dbContext = dbContext;
     }
 
     [HttpPost]
-    public async Task<IActionResult> CreateProduct([FromBody] CreateProductDto productdto)
+    public async Task<IActionResult> CreateProduct([FromBody] CreateProductDto productdto, CancellationToken cancellationToken = default)
```
Comment (wahid-d) [inline Controllers/ProductsController.cs:22]:
```suggestion
    public async Task<IActionResult> CreateProduct(
        [FromBody] CreateProductDto productdto, 
        CancellationToken cancellationToken = default)
```

Comment (wahid-d) [inline Controllers/ProductsController.cs:22]: why add this?

File: Controllers/ProductsController.cs [hunk 4]
```diff
@@ -34,7 +34,7 @@ public async Task<IActionResult> CreateProduct([FromBody] CreateProductDto produ
             CategoryId = productdto.CategoryId
         });
 
-        await dbContext.SaveChangesAsync();
+        await dbContext.SaveChangesAsync(cancellationToken);
```
Comment (wahid-d) [inline Controllers/ProductsController.cs:37]: why?

**Review by wahid-d — CHANGES_REQUESTED** [review]
.idea degan papkadagi o'zgarishlarni chopish kerak.

**Review by wahid-d — APPROVED** [review]

**Review by sarvar-swe — APPROVED** [review]

**Pull request merged**